include/mxnet/ndarray.h

```diff
@@ -779,6 +783,12 @@ class NDArray {
   void UpdateMKLDNNMemDesc();
 #endif
 
+#if MXNET_USE_NGRAPH == 1
+  // create and return tensor_view with this ndarray mem
+  std::shared_ptr<ngraph::runtime::Tensor> &create_tensor(bool is_boolean,
+                                                          bool is_scalar);
```

Comment:
@mbrookhart after some discussion with @szha , we would really prefer to not include ngraph in NDArray because NDArray is a class exposed to C++ users. We consider it's part of our API. Changing API is always a big deal for us. The subgraph API actually provides a way to hide everything inside nGraph. I would like to suggest changing your mxnet bridge so that you don't need to put nGraph stuff in NDArray.

Reply:
Removed the NDArray changes and validated performance for CPU, other backends will come in a future PR.